### DISCUSSION TITLE
Explicitly Open or Close a pane or a dock (rather than toggle)

### DISCUSSION THREAD

**Discussion by pjv** [zed-industries/zed#12848] [Feature Requests]
I have some trackpad gestures that let me move Zed around on my wide screen. 3-finger-swipe-left sends it to the left side of the screen, 3-finger-swipe-right to the right side. 3-finger-double-tap sends it to the middle and makes it a bit wider than the other two.

What I want to add to these gestures is the ability to open the assistant panel when the window is centered (3-finger-double-tap) and close the assistant panel for the other two. But with the only command / key binding available being “toggle”, it doesn’t work in all cases. For example, if I start with the window on the left and no assistant panel showing and I swipe it to the right, it toggles the assistant panel open which I don’t want.

Is there any way to have commands that explicitly open or close those docks / panels rather than toggling them to whatever state they are not currently in?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
